app/controllers/films_controller.rb

```diff
@@ -16,9 +16,16 @@ def show
 
   def new
     @film = Film.new
+    @film_search = OmdbService.new(params[:search], params[:year])
+    return unless @film_search
+
+    @film.title = @film_search.search.parsed_response['Title']
+    @film.description = @film_search.search.parsed_response['Plot']
   end
 
-  def edit; end
+  def edit
+    @film_search = OmdbService.new(params[:search], params[:year])
```

Comment:
OmdbService called twice - in .new and .edit. How about moving this to a separate method?

Reply:
And also - what if one of these parameters is missing?